nats-top.yaml

```diff
@@ -51,3 +51,4 @@ test:
         sleep 2
         nats-top -o nats-top-output
         cat nats-top-output | grep -Ei 'server|bytes|sec'
+    - uses: go/covdata
```

Comment:
It requires:
- binary to be built with `go build -cover`
- the `GOCOVERDIR` set (unfortunate, as currently pipelines allows to inject packages but not environment to the test environment)